### PR TITLE
Display outfits

### PR DESCRIPTION
**Pull request by happysituations**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -59,7 +59,7 @@ const client = {
     publicPath: '/'
   },
   module: moduleObj,
- plugins: [htmlWebpackPlugin, new ExtractTextPlugin("styles.css")]
+  plugins: [htmlWebpackPlugin, new ExtractTextPlugin("styles.css")]
```
Comment (skoshkarli) [inline webpack.config.js]: extra indent

File: src/server/index.js [hunk 2]
```diff
@@ -196,11 +196,19 @@ app.get('/getitems', (req, res) => {
...
+    userId: req.query.userId, // need to update based on how the user is stored
+  };
+  db.getOutfits(data, (result, error) => {
+
 app.post('/edititem', (req, res) => {
   let data = {
     userId: req.query.userId, // need to update based on how the user is stored
   };
   db.editItem(req.body, (result, error) => {
+
```
Comment (skoshkarli) [inline src/server/index.js]: unnecessary newline

File: src/server/index.js [hunk 3]
```diff
@@ -196,11 +196,19 @@ app.get('/getitems', (req, res) => {
     }});
 });
 
+
+app.get('/getoutfits', (req, res) => {
+  let data = {
+    userId: req.query.userId, // need to update based on how the user is stored
+  };
+  db.getOutfits(data, (result, error) => {
+
```
Comment (skoshkarli) [inline src/server/index.js]: weird

File: src/client/components/common/OutfitsContainer/Outfit.jsx [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+const mapStateToProps = state => {
+  return {
+    allItemsArray: state.closet.allItemsArray
+  };
+};
+
+const mapDispatchToProps = dispatch => ({});
+
+export default withRouter(
+  connect(
```
Comment (skoshkarli) [inline src/client/components/common/OutfitsContainer/Outfit.jsx]: can we one line this

File: src/client/components/common/OutfitsContainer/Outfit.jsx [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+      }
+    }
+    return {};
+  }
+
+  render() {
+    let items = (
+      <Grid>
+        {this.props.itemids.map(id => {
+          if (id === null) return '';
```
Comment (skoshkarli) [inline src/client/components/common/OutfitsContainer/Outfit.jsx]: should return '' or return null ?

File: src/client/components/MyCloset/MyCloset.jsx [hunk 6]
```diff
@@ -49,7 +50,9 @@ export class MyCloset extends React.Component {
               />
             </Grid.Column>
             <Grid.Column mobile={16} computer={16} tablet={16} widescreen={13} largeScreen={13}>
-              <MyClosetItemsContainer/>
+              {this.props.currentMenuItem.includes('Outfit')
```
Comment (skoshkarli) [inline src/client/components/MyCloset/MyCloset.jsx]: this is cool for now, but could be dangerous down the line when/if we allow someone to add a closet and name it.

File: src/client/actions/myClosetActions.js [hunk 7]
```diff
@@ -72,11 +73,20 @@ export function updatedModalState(isModalDisplayed, item) {
...
+  };
+}  
+  
 export function updateDropdownInfo(colors, categories, styles) {
   return {
     type: ALL_DROPDOWN_INFO,
     colors,
     categories,
     styles
+
```
Comment (skoshkarli) [inline src/client/actions/myClosetActions.js]: unnecessary new line

File: webpack.config.js (hunk 1 above)
Comment (happysituations) [reply webpack.config.js]: fixed

File: src/client/components/common/OutfitsContainer/Outfit.jsx (hunk 4 above)
Comment (happysituations) [reply src/client/components/common/OutfitsContainer/Outfit.jsx]: updated

File: src/client/components/common/OutfitsContainer/Outfit.jsx (hunk 5 above)
Comment (happysituations) [reply src/client/components/common/OutfitsContainer/Outfit.jsx]: ''

File: src/client/components/MyCloset/MyCloset.jsx (hunk 6 above)
Comment (happysituations) [reply src/client/components/MyCloset/MyCloset.jsx]: agreed

File: src/database/index.js [hunk 8]
```diff
@@ -324,6 +324,25 @@ const dbHelpers = {
       cb(organizedData);
     });
   },
+
+  //return array of outfits
+  getOutfits: (data, cb) => {
+
```
Comment (skoshkarli) [inline src/database/index.js]: empty line

**Pull request merged**